SimpleSDMPolygons/src/keychecker.jl

```diff
@@ -0,0 +1,21 @@
+"""
+    keychecker
+"""
+function keychecker(data::P; kwargs...) where {P<:PolygonData}
+    # Check for levels
+    if :level in keys(kwargs)
+        isnothing(levels(data)) && error("The $(P) dataset does not allow for month as a keyword argument")
```

Comment:
the error message should say `levels` not `months`

Reply:
done